docs/Makefile

```diff
@@ -55,8 +55,3 @@ linkcheck:
 		echo "Running linkcheck in $$i..."; \
 		(cd $$i; $(MAKE) $(MFLAGS) $(MYMAKEFLAGS) linkcheck); done
 
-gallerytest:
```

Comment:
I am suggesting running the tests directly with `pytest`, because that also allows you to test individual examples (see dev guide change).